[1,"Error occurred during cleanup: Error: El proceso no tiene acceso al archivo porque est√° siendo utilizado por otro proceso. (os error 32): remove 'C:\\Users\\niug9\\AppData\\Local\\Temp\\quarto-sessiondfb799b92efb0be1'\n"]
